src/System.Management.Automation/engine/parser/TypeInferenceVisitor.cs

```diff
@@ -564,23 +564,23 @@ object ICustomAstVisitor.VisitHashtable(HashtableAst hashtableAst)
                             switch (expression)
                             {
                                 case ConstantExpressionAst constantExpression:
-                                {
-                                    value = constantExpression.Value;
-                                    break;
-                                }
+                                    {
```

Comment:
indenting the brace seems odd to me, perhaps just remove the brace

Reply:
Fixed.